modules/telegramBot/telegram.go

```diff
@@ -0,0 +1,35 @@
+package telegramBot
+
+import (
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api/v5"
+	"log"
+)
+
+const (
+	defaultTimeout = 60
+)
+
+func initTelegram(cfg *TelegramBot) *TelegramBotImpl {
+	bot, err := tgbotapi.NewBotAPI(cfg.ApiToken)
+
+	if err != nil {
+		log.Panic(err)
+	}
+
+	return &TelegramBotImpl{Bot: bot}
+}
+
+func (t *TelegramBotImpl) ListenForMessage(fn func(message tgbotapi.Update)) {
+	u := tgbotapi.NewUpdate(0)
+	u.Timeout = defaultTimeout
```

Comment:
```suggestion
	u := tgbotapi.UpdateConfig{
		Offset:  0,
		Limit:   0,
		Timeout: defaultTimeout,
	}
```
Лучше создать конфиг сразу с нужной конфигурацией, чем потом руками менять поля